package.json

```diff
@@ -34,6 +35,7 @@
     ]
   },
   "devDependencies": {
-    "react-bootstrap": "^1.0.1"
+    "react-bootstrap": "^1.0.1",
+    "react-test-renderer": "^16.13.1"
```

Comment:
react testing library should suffice. I'm not sure why we need this

Reply:
I was just following the example you gave me from work which is why I thought it would be acceptable ;P